src/rpc/RPCHelpers.cpp

```diff
@@ -1539,47 +1539,51 @@ traverseTransactions(
     }
 
     auto minIndex = context.range.minSequence;
+    auto maxIndex = context.range.maxSequence;
+    std::optional<int64_t> min;
+    std::optional<int64_t> max;
+
     if (request.contains(JS(ledger_index_min)))
     {
-        auto& min = request.at(JS(ledger_index_min));
+        if (request.at(JS(ledger_index_min)).is_int64())
```

Comment:
I'd suggest to simplify this code to something like 
```
if (not req.at(index_min).is_int64()) 
    return error;
min = req.at(index_min).as_int64();
if (*min != -1) { 
    ... 
}
```
And the same transformation for `max` below.